app/models/membership.rb

```diff
@@ -22,7 +22,7 @@ def leader?
   end
 
   def newbie?
-    has_post?(DEFAULT_POST_ID)
+    has_post?(DEFAULT_POST_ID) && self.end.nil? && !self.archived?
```

Comment:
Here we can leave the `self`, it will result cleaner code.

Reply:
You're rigth. I was thinking about it, but the `end` is a reserved key word. 
Maybe the solution is if we use nicer naming, example: end_date end start_date instead of end and start.